cla-backend/cla/utils.py

```
@@ -1748,8 +1742,8 @@ def lookup_gitlab_org_members(organization_id):
         r = requests.get(f"{cla.config.PLATFORM_GATEWAY_URL}/cla-service/v4/gitlab/group/{organization_id}/members")
         r.raise_for_status()
     except requests.exceptions.HTTPError as err:
-        cla.log.warning(f"Could not fetch gitlab org users: {err}")
-        return {f"error: Could not get user gitlab group id: {organization_id} members: {err}"}
+        cla.log.warning(f"Could not fetch gitlab org users: API error occurred")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)